lib/LightPanel/package.json

```diff
@@ -1,6 +0,0 @@
-{
-	"main": "LightPanel.js",
-	"styles": [
-		"LightPanel.less"
```

Comment:
This isn't included in the build any longer because it's not referenced by LightPanels/package.json